ember-power-select/src/helpers/ember-power-select-is-group.ts

```diff
@@ -1,8 +1,7 @@
-import { helper } from '@ember/component/helper';
 import { isGroup } from '../utils/group-utils.ts';
 
-export function emberPowerSelectIsGroup([maybeGroup]: [any]): boolean {
+export function emberPowerSelectIsGroup(maybeGroup: any): boolean {
```

Comment:
here we could just now call `isGroup` directly when we use it (just 1 place) and remove this helper. same with the `isNone`, let me know if we want to do that change.